.github/workflows/ci.yml

```diff
@@ -35,16 +35,35 @@ jobs:
           REQUIRE_DEV: true
 
   phpunit:
-    name: "PHPUnit on ${{ matrix.php }} ${{ matrix.phar && '(with phar)' || '' }}"
+    name: "PHPUnit on ${{ matrix.php }} | Castor from ${{ matrix.castor.method }}"
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        php: [ "8.1", "8.2", "8.3" ]
-        phar: [ false ]
         include:
           - php: "8.1"
-            phar: true
+            castor:
+              bin: 'bin/castor'
+              method: 'vendor'
```

Comment:
I don't really like "vendor". But I haven't a better proposal 🙈 
may be "php bin/castor"?